src/components/Navbar/NavigationDrawer/NavigationDrawerBody/NavigationDrawerBody.module.scss

```diff
@@ -17,5 +46,44 @@
 
 .projectsDesc {
   font-size: var(--font-size-small);
-  margin-block-start: var(--spacing-small);
+  margin-block-end: var(--spacing-small);
+}
+
+.accordionHeaderLeft {
+  gap: var(--spacing-xxsmall);
+  align-items: center !important;
+}
+
+.accordionContent {
+  margin-inline-start: var(--spacing-mega);
```

Comment:
For all new css, we should use pixel values not rem as per agreenment with Talha to make sure what is on Figma is 100% the same pixel values in the code base. We already have pixel values suffixed with `-px` e.g. `--spacing-xxsmall-px` etc. Please apply to entire file.